spec/models/searchgov_url_spec.rb

```diff
@@ -125,6 +130,17 @@
         fetch
       end
 
+      context 'when the record is enqueued for reindex' do
+        let(:searchgov_url) do
+          SearchgovUrl.create!(valid_attributes.merge(enqueued_for_reindex: true))
+        end
+
+        it 'sets enqueued_for_reindex to false' do
+          expect{ fetch }.to change{ searchgov_url.enqueued_for_reindex }.
```

Comment:
indent 2 instead of 4 spaces

Reply:
bump

Reply:
Not sure where you see 4 spaces.

Reply:
Weird - I must have commented on the wrong line. I do still still 4 instead of 2 spaces in a number of places throughout the `fetchable_behavior` spec.